doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -612,6 +612,12 @@ julia> 2(x-1)^2 - 3(x-1) + 1
 3
 ```
 
+!!! warning
+    Notice that, in this case, the precedence of numeric literal coefficient is
+    still the same as that of the unary operators. Hence, the expression
+    `6/2(2+1)` leads to `1` instead of the conventional result `9`, which is
```

Comment:
To me, this reads like a direct copy of the previous text in the manual. Perhaps instead it would be better for the text here to give clear guidance on what makes it different, rather than reiterating what it is the same as:
```
!!! note

The precedence of numeric literal coefficients used for implicit multiplication is higher
than other binary operators such as multiplication (`*`), division (`/`, `\`, and `//`), and exponentiation (`^`).
This means, for example, that `1 / 2im` equal `-0.5im` and `6 // 2(2 + 1)` equals `1 // 1`.
```